Assets/HotmaxEditorVR/stateManager.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class stateManager : MonoBehaviour
+{
+    public static editorModes editorMode = editorModes.universalTransformMode;
+    public enum editorModes
+    {
+        universalTransformMode,
+        cloneDeleteMode,
+        openMenuMode
+    }
+    public delegate void editorModeHandler(editorModes value);
+    public static event editorModeHandler editorModeEvent;
+
+    public static bool rotationGizmoIsSelected = false;
```

Comment:
can you clean up these spaces, make sure everything in the top is grouped together